Platform/Microsoft/Application/AuthVarTest/AuthVarTest.c

```diff
@@ -137,17 +137,18 @@ VerifyVariableEnumerable (
       VERIFY_IS_NOT_NULL (mEnumVariableName);
       VariableNameBufferSize = VariableNameSize;
 
-      VERIFY_SUCCEEDED (gRT->GetNextVariableName (
-                              &VariableNameBufferSize,
-                              mEnumVariableName,
-                              &VendorGuid)
-                              );
-    } else if (Status == EFI_NOT_FOUND) {
+      Status = gRT->GetNextVariableName (
+                    &VariableNameSize,
+                    mEnumVariableName,
+                    &VendorGuid
+                    );
+    }
+
+    if (Status == EFI_NOT_FOUND) {
       // Reached end of variable list.
       break;
     } else {
-      VERIFY_SUCCEEDED (Status, "Verify variable retrieval");
-
+      VERIFY_SUCCEEDED (Status, "Verify variable retrieval");      

```

Comment:
Nit - No trailing whitespace

```suggestion
      VERIFY_SUCCEEDED (Status, "Verify variable retrieval");
```